src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,8 +1,47 @@
 package pl.javastart.task;
 
+import java.util.Scanner;
+
 public class Main {
 
+    public static final int MAX_MEMORY = 3;
+
     public static void main(String[] args) {
 
+        Scanner sc = new Scanner(System.in);
+
+        Memory[] memories = new Memory[MAX_MEMORY];
+
+        int memorycount = 0;
+
+        System.out.println("Podaj informację o trzech unikalnych pamięciach RAM");
+
```

Comment:
Nie dodawaj tych enterów do rozdzielania kodu. Jeżeli w metodzie pojawia się enter, to zazwyczaj sugeruje to, że coś można wydzielić do osobnej metody